__tests__/pet.test.js

```diff
@@ -0,0 +1,153 @@
+const Pet = require("../src/pet.js");
+describe("constructor", () => {
```

Comment:
Just adding a clear line here to separate the imports from the start of the tests
```
const Pet = require("../src/pet.js");

describe("constructor", () => {
```

Also, if you look at the test output in the terminal, there is a lot of unnecessary indentation caused by nesting an awful lot of describe blocks where you dont need to. These should be used to group relevant tests together, and a lot of these could be removed.

An example structure could be:

```
describe("constructor", () => {
  // all it block tests relating to the constructor
});

describe("Grow up", () => {
  // all it block tests relating to the growUp method
});
```
etc. Avoid all the nesting and will give you a cleaner output